release_images.yml

```diff
@@ -2,27 +2,26 @@
 release_images:
   1:
     framework: "tensorflow"
-    version: "2.2.0"
+    version: "2.0.2"
     training:
-      device_types: ["gpu"]
-      python_versions: ["py37"]
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py27", "py36"]
       os_version: "ubuntu18.04"
-      cuda_version: "cu102"
-      dlc_version: "2.0"
-      disable_sm_tag: True
+      cuda_version: "cu100"
+      dlc_version: "5.2"
     inference:
       device_types: ["cpu", "gpu"]
-      python_versions: ["py37"]
+      python_versions: ["py27", "py36"]
       os_version: "ubuntu18.04"
-      cuda_version: "cu102"
-      dlc_version: "1.0"
+      cuda_version: "cu100"
+      dlc_version: "5.2"
   2:
     framework: "tensorflow"
-    version: "2.2.0"
+    version: "2.0.2"
     training:
       device_types: ["gpu"]
-      python_versions: ["py37"]
+      python_versions: ["py27", "py36"]
```

Comment:
I believe this should be only "py36". Please get confirmation if this is correct.